packages/javascript-sdk/src/http-client/helpers.ts

```diff
@@ -120,6 +120,11 @@ export function examineForIGAuthz(res: Response): boolean {
   return type.includes('html') && res.url.includes('composite_advice');
 }
 
+export function examineForIGAuthzHeader(res: Response): boolean {
+  const authnHeader = res.headers.get('WWW-Authenticate') || '';
+  return authnHeader.includes('advices');
+}
+
```

Comment:
Much of the code in this file written for the PoC, seemed to work nicely. I need to add tests for the helpers, likely tomorrow.